package.json

```diff
@@ -75,7 +75,8 @@
     "classnames": "^2.2.3",
     "color": "^1.0.3",
     "lodash": "^4.17.4",
-    "react-transition-group": "^2.2.0"
+    "react-router-dom": "^4.2.2",
```

Comment:
Could we make this something that the user sends in instead? (Would like to keep out "specialized" dependencies such as this from the ui-kit and keep it general)

Reply:
I understand, but I don't see an easy solution for handling the states of each <PopupMenuItem /> without using <Link />. We would move a lot of logic to the component implementing <PopupMenu /> and bloat those instead.